lnrpc/walletunlocker.swagger.json

```diff
@@ -0,0 +1,305 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "walletunlocker.proto",
+    "version": "version not set"
+  },
+  "schemes": [
+    "http",
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
```

Comment:
So I think projects like BTCPay may depend on the fact that rn, there's only a single swagger file they need to consume in order to generate their HTTP client code. As a result, we either need to keep them as a single file, or notify the consumers, as this may essentially be a breaking change for them.

Reply:
Looks like they've only updated the swagger file once since it's been created: https://github.com/btcpayserver/BTCPayServer.Lightning/commits/121f70c10da523ef751058d5281196935a8c5c6f/src/BTCPayServer.Lightning.LND/LndSwaggerClient.cs

Reply:
So I think things are OK, assuming they just keep that same generated code (like they have for the past two years). If it turns out to be an issue later, we can revert, and also flag their attention on this early.